settings

```diff
@@ -1,32 +1,51 @@
 VERSION="${VERSION:-1.23}"
+PROJECT="${PROJECT:-foreman}"
+PROJECTDIR="releases/$PROJECT"
+RELEASEDIR="${PROJECTDIR}/${VERSION}"
+
+# Prepare settings file
+if [[ ! -d "$RELEASEDIR" ]] ; then
+	mkdir -p "$RELEASEDIR"
+	echo "FULLVERSION='${VERSION}.0-RC1'" > "$RELEASEDIR/settings"
+fi
+
+if [[ $PROJECT != foreman ]]; then
+  . "$RELEASEDIR/settings"
+  if [[ -z "$FOREMAN_VERSION" ]]; then
+    echo "please set FOREMAN_VERSION in $RELEASEDIR/settings"
+    exit 1
+  fi
+else
+  FOREMAN_VERSION="$VERSION"
```

Comment:
given we share data with foreman, I wonder if we should load the foreman settings here? esp the key definitions are duplicated now

Reply:
That was exactly what I was thinking.